config/crd/bases/nvidia.com_clusterpolicies.yaml

```diff
@@ -136,13 +136,14 @@ spec:
                 properties:
                   default:
                     default: false
-                    description: Default indicates whether to use CDI as the default
-                      mechanism for providing GPU access to containers.
+                    description: 'Deprecated: This field is no longer used. Setting
+                      cdi.enabled=true will configure CDI as the default mechanism
+                      for making GPUs accessible to containers.'
                     type: boolean
                   enabled:
-                    default: false
-                    description: Enabled indicates whether CDI can be used to make
-                      GPUs accessible to containers.
+                    default: true
+                    description: Enabled indicates whether CDI should be used as the
```

Comment:
we should probably expand CDI here. Let's update the godoc in `clusterpolicy_types.go`